lib/phoenix/channel.ex

```diff
@@ -124,17 +124,34 @@ defmodule Phoenix.Channel do
                                                                      {:leave, Socket.t} |
                                                                      {:error, reason :: term, Socket.t}
 
-  defmacro __using__(_options) do
+  defmacro __using__(options \\ []) do
     quote do
       @behaviour unquote(__MODULE__)
-      import unquote(__MODULE__)
+      @pubsub_server unquote(options[:pubsub_server])
```

Comment:
Yeah I was planning on inflecting it from the app name. Like `:my_app` -> `:my_app_pub`. It would also match the name that we generate/pass in the generated supervision tree.